src/accountsdb/snapshots.zig

```diff
@@ -15,7 +15,7 @@ const Slot = @import("../core/time.zig").Slot;
 const Epoch = @import("../core/time.zig").Epoch;
 const Pubkey = @import("../core/pubkey.zig").Pubkey;
 const bincode = @import("../bincode/bincode.zig");
-const defaultArrayListOnEOFConfig = @import("../utils/arraylist.zig").defaultArrayListOnEOFConfig;
```

Comment:
for any new imports were following the convention below (see `cmd.zig`)
```
const sig = @import("../lib.zig")
const X = sig.bincode.arraylist.X;
```
we only recently decided this and are slowly moving over imports to follow this - could you change these imports to follow this format too?